components/expenses/PayoutMethodData.js

```diff
@@ -9,6 +9,9 @@ import Container from '../Container';
 
 const renderObject = object =>
   Object.entries(object).reduce((acc, [key, value]) => {
+    if (typeof value === 'object') {
+      return [...acc, ...renderObject(value)];
+    }
```

Comment:
Is this still necessary given that we now have a special display for `details`? https://github.com/opencollective/opencollective-frontend/blob/890f5bc63f2b1d605dc547a279634b1834e972f4/components/expenses/PayoutMethodData.js#L85

Reply:
Oh for the `location` right?

Reply:
I realize that I forgot to add the reviewers on https://github.com/opencollective/opencollective-frontend/pull/3947 - where this was broken. Thanks for fixing this so quickly :+1:

Reply:
@Betree, yes, for the address! No problem, thanks for confirming, I had a strange feeling I had implemented this fix before. 😂